docs/rules/eol-last.md

```diff
@@ -1,4 +1,4 @@
-# Require file to end with single newline (eol-last)
+# Require or disallow file to end with single newline (eol-last)
```

Comment:
I believe we are making these titles start with a lowercase character now - can someone else on the team  confirm?